server/src/controllers/auth.mts

```diff
@@ -100,6 +100,26 @@ export const authenticateGoogleUser = async (req: Request, res: Response, next:
   }
 };
```

Comment:
Is this still being needed after being replaced in the route?

Reply:
No, the old middleware isn't necessary if it isn't being used. I was perhaps over cautious so as to not delete anything that you might have needed.

Reply:
All good, you can probably delete it since i can take it off a previous commit if i really needed to reference it; same goes for any other future similar cases. I'll request a change and after it's deleted it can be merged asap.

Reply:
Deleted the outdated code. Had to do some merging and rebasing to squash commits.